ubuntudesign/www.ubuntu.com
# Replace native search solution with ubuntudesign.gsa

This has required a small restructure of search.html, and now introduces new system dependencies (libxml2-dev etc.) but fixes many issues:

- Special characters display properly
- Search parsing errors are history
- We have the option to filter by domain or language

Resolves #1263.

**NB:** This should be reviewed but not merged until we're ready, as it requires a spec change to add `lxml` system dependencies.

QA
---

*Connect to the VPN* and run the site natively:

``` bash
docker-compose run sass
virtualenv env
env/bin/pip install -r requirements/dev.txt
env/bin/python manage.py runserver 0.0.0.0:8001
```

Now visit <http://127.0.0.1:8001/search> and check it looks the same as live, but works better (special characters etc.).
